File: Backend/src/main/java/com/devsrcoppel/crud/models/Empleado.java
package com.devsrcoppel.crud.models;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Empleado {
    @JsonProperty("IdEmpleado")
    private String IdEmpleado;
    @JsonProperty("Nombre")
    private String Nombre;
    @JsonProperty("Apellido")
    private String Apellido;
    @JsonProperty("IdPuesto")
    private String IdPuesto;
    @JsonProperty("Contraseña")
    private String Contraseña;
    @JsonProperty("Token")
    private String Token;

    public Empleado() {

    }
}

File: Frontend/CRUD/Modelos/Empleado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CRUD.Modelos
{
    /// <summary>
    /// Clase para los datos de empleado.
    /// </summary>
    public class Empleado
    {
        public String IdEmpleado { get; set; }
        public String Nombre { get; set; }
        public String Apellido { get; set; }
        public String? IdPuesto { get; set; }
        public String Contraseña { get; set; }
        public String Token { get; set; }
    }
}
